modules/caddytls/certmanagers.go

```diff
@@ -22,6 +22,9 @@ func init() {
 	caddy.RegisterModule(HTTPCertGetter{})
 }
 
+// For referencing the requested SNI server name.
+var SNIServerNameCtxKey caddy.CtxKey = "sni_server_name"
```

Comment:
```suggestion
const ClientHelloSNICtxKey caddy.CtxKey = "client_hello_sni"
```

This is potentially bikeshedding but hear me out

Reply:
I agree; `SNIServerName` is basically "server name indication server name" 😅 https://en.m.wikipedia.org/wiki/Server_Name_Indication

Reply:
lol, I didn't even think about what the expanding name would be.  Indeed, that sounds pretty silly :) Updated name coming up.

Reply:
done